### DISCUSSION TITLE
How to delete ran tasks ?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [triggerdotdev/trigger.dev#1176] [Q&A]
I don't see any options to delete ran tasks, I think we need an option to delete ran task info or delete after xx days.
![image](https://github.com/triggerdotdev/trigger.dev/assets/15695252/493dcc22-ae0f-4595-8cc0-aa59f8d98898)

**Comment by samuelbohl** [comment]
Also very important for us. Self hosting often comes with some storage limits.

**Comment by lpkobamn** [comment]
#1567
